app/src/main/java/com/pps/globant/fittracker/mvp/view/LoginView.java

```diff
@@ -107,4 +111,37 @@ public void popUp(@StringRes int error) {
     public class FbButtonPressedEvent {
     }
 
+    /********instagram*******/
+
+    public void setIgTextView(String name){
+        igLabel.setText(name);
+    }
+
+    public void showLoginToast(String name){
+        Toast.makeText(this.getContext(), getContext().getResources().getString(R.string.toast_instagram_login) + name, Toast.LENGTH_SHORT).show();
```

Comment:
It is necessary use "getContext().getResources().getString(R.string.toast_instagram_login)" isn't work with "R.string.toast_instagram_login"??

Reply:
Yes! If i didnt use "getContext().getResources().getString(R.string.toast_instagram_login)" when the toast was shown it contains a number. This was the only case that i had to use it in that way, so i think its because of "+ name "

Reply:
ok np

Reply:
You need use String.format instead the +

Reply:
done!